plaid/api/deposit_switch.py

```diff
@@ -0,0 +1,47 @@
+from plaid.api.api import API
+
+
+class DepositSwitch(API):
+    '''
+    Deposit Switch endpoints.
+    (`HTTP docs <coming soon>`__)
+    '''
+
+    def get(self, deposit_switch_id):
+        '''
+        Gets deposit switch information given a deposit switch id.
+
+        :param  str deposit_switch_id:  ID of deposit switch to get.
+        '''
+
+        return self.client.post('deposit_switch/get', {
```

Comment:
Should we be using `/deposit_switch/get` or `deposit_switch/get` for sake of consistency? I see both patterns throughout the PR